app/scripts/Annotations1dTrack.js

```diff
@@ -3,21 +3,23 @@ import * as PIXI from 'pixi.js';
 
 import PixiTrack from './PixiTrack';
 
+import pubSub from './services/pub-sub';
+
+const MAX_CLICK_DELAY = 300;
```

Comment:
What's this for?

Reply:
I though I replied but maybe I didn't.

This is used for distinguishing between a canceled click (long click) and a normal click.

Reply:
```suggestion
// for distinguishing between a cancelled click (long click) and
// a normal click
const MAX_CLICK_DELAY = 300;
```
More comments, more better.